pom.xml

```diff
@@ -640,6 +639,46 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>error-prone-9+</id>
+      <activation>
+        <jdk>[9,)</jdk>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <executions>
+              <execution>
+                <!-- only use errorprone on main source tree -->
+                <id>default-compile</id>
+                <phase>compile</phase>
+                <goals>
+                  <goal>compile</goal>
+                </goals>
+                <configuration>
+                  <forceJavacCompilerUse>true</forceJavacCompilerUse>
+                  <compilerArgs>
+                    <arg>-XDcompilePolicy=simple</arg>
+                    <arg>-Xplugin:ErrorProne</arg>
+                    <!-- TODO: why does this not work? -->
+                    <!--<arg>-XepDisableWarningsInGeneratedCode</arg>-->
```

Comment:
this crashes when uncommented:

```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] javac: invalid flag: -XepDisableWarningsInGeneratedCode
Usage: javac <options> <source files>
use --help for a list of possible options
warning: [options] bootstrap class path not set in conjunction with -source 1.8
```

Reply:
You need to put all the Error Prone flags in the same `<arg>` as `-Xplugin:ErrorProne` like
```
<arg>-Xplugin:ErrorProne -XepDisableWarningsInGeneratedCode</arg>
```

Here's an example from another project I work on that uses Error Prone as a compiler plugin:  https://github.com/TechEmpower/tfb-status/blob/e3e1c5d2684252d2ad585545016c98574bf78e81/pom.xml#L451-L534